controllers/administration.js

```diff
@@ -748,13 +748,23 @@ const returnAdminPrefix = (roles) => {
 };
 
 // with userId to accountId
-const userIdtoAccountIdUpdate = (service) => function useIdtoAccountId(req, res, next) {
+const userIdToAccountIdUpdate = (service) => async function useIdtoAccountId(req, res, next) {
```

Comment:
please remove (service) and write 'accounts' in line 763

Reply:
:+1: